### PR TITLE
MRG: Fix for old mayavi

### LINKED ISSUE #4695: MNE Coreg Exception in traits notification handler
In python 2.7 conda venv `mne coreg` fails to load digitized head points due to 

> Exception occurred in traits notification handler.
> Please check the log file for details.
> Exception occurred in traits notification handler for object: <mne.gui._viewer.PointObject object at 0x7f79eafd7590>, trait: scene, old value: <mayavi.tools.mlab_scene_model.MlabSceneModel object at 0x7f79eb1b6170>, new value: <mayavi.tools.mlab_scene_model.MlabSceneModel object at 0x7f79eb46cf50>
> Traceback (most recent call last):
>   File "/home/ktavabi/miniconda3/envs/andrea-anders/lib/python2.7/site-packages/traits/trait_notifiers.py", line 519, in _dispatch_change_event
>     self.dispatch( handler, *args )
>   File "/home/ktavabi/miniconda3/envs/andrea-anders/lib/python2.7/site-packages/traits/trait_notifiers.py", line 482, in dispatch
>     handler( *args )
>   File "/home/ktavabi/Projects/mne-python/mne/gui/_viewer.py", line 225, in _plot_points
>     if not self.scene.running():
> AttributeError: 'MlabSceneModel' object has no attribute 'running'
> 

cc @larsoner 

image

### PR DESCRIPTION
**Pull request by larsoner**
Closes #4695.

@ktavabi can you see if this fixes things for you?

### CODE REVIEW

**Comment by larsoner** [comment]
It fails on this PR but passes on an earlier version of MNE (e.g., 0.14)? That's odd because it seems like the GLX context creation is a basic/fundamental VTK thing that we haven't changed.

**Comment by ktavabi** [comment]
Odds of me trying this branch immediately after system update. 100%
Sorry dude. I had to restart. Pretty sure this is working now!

**Comment by agramfort** [comment]
@larsoner all green with CIs are

**Comment by larsoner** [comment]
This fix isn't quite right, will need to make some changes

**Comment by larsoner** [comment]
ready for review/merge from my end

File: mne/gui/tests/test_kit2fiff_gui.py [hunk 1]
```diff
@@ -131,25 +134,32 @@ def test_kit2fiff_gui():
...
+        frame.model.stim_threshold = 10.
+        frame.model.stim_chs = 'save this!'
+        frame.save_config(home_dir)
+        ui.dispose()
+
+        gui.process_events()
+
+        # test setting persistence
+        ui, frame = mne.gui.kit2fiff()
+        return
```
Comment (agramfort) [inline mne/gui/tests/test_kit2fiff_gui.py]: ?

**Comment by larsoner** [comment]
Removed debugging line

**Comment by agramfort** [comment]
thx @larsoner

**Comment by agramfort** [comment]
backported.

**Pull request merged**